site/src/sphinx/server-annotated-service.rst

```diff
@@ -190,6 +190,36 @@ In this case the variable name is used as the value.
         public HttpResponse hello1(@Param String name) { ... }
     }
 
+.. note::
+
+    You can configure ``-parameters`` javac option with your build framework as follows.
```

Comment:
How about:

    You can configure your build tool to add ``-parameters`` javac option as follows.